src/test/java/org/mifos/integrationtest/cucumber/stepdef/VoucherManagementStepDef.java

```diff
@@ -396,9 +394,11 @@ public void redeemVoucherFailureUsingAnnotation() {
 
     @Then("I check for redeem voucher failure")
     public void redeemVoucherFailure() {
-        iCanCreateAnRedeemVoucherRequestDTOForVoucherRedemption();
-        iCallTheRedeemVoucherAPIWithExpectedStatusOf(200);
-        assertUnsuccessfulRedemption();
+        await().atMost(awaitMost, SECONDS).pollDelay(pollDelay, SECONDS).pollInterval(pollInterval, SECONDS).untilAsserted(() -> {
```

Comment:
here as well, Await doesn't make much sense as we are making a post request. repeating this step has no value.